qml/keys/SymbolShiftKey.qml

```diff
@@ -26,7 +26,7 @@ ActionKey {
     shifted: "+=[]";
     action: "symbols";
 
-    fontSize: UI.symbolShiftKeyFontSize;
+    fontSize: UI.xsFontSize;
```

Comment:
it's a bit weird to not use UI.symbolShiftKeyFontSize for the SymbolShiftKey font size... Is UI.symbolShiftKeyFontSize used elsewhere now ? Shouldn't we prefer simply changing symbolShiftKeyFontSize's value ?